devops/charts/server/templates/routes.yaml

```diff
@@ -0,0 +1,20 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: {{include "vdr-proxy.fullname" .}}
+  labels: {{- include "vdr-proxy.labels" . | nindent 4}}
+  annotations: {{- toYaml .Values.route.annotations | nindent 4}}
+spec:
+  {{- if .Values.route.host }}
+  host: {{ .Values.route.host }}
+  {{- end }}
+  to:
+    kind: Service
+    name: {{include "vdr-proxy.fullname" .}}
+    weight: 100
+  port:
+    targetPort: {{.Values.service.targetPort}}
+  tls:
```

Comment:
It's the BCGov policy to not use OCP routes for public facing resources. They prefer you get a domain and your own SSL certificate. Able to bring this up with Clecio. Because its machine-2-machine it might be fine but he should make that call.

Reply:
I mean public facing production resources. It's fine for dev and test.